src/main/java/cyclops/stream/ReactiveSeq.java

```diff
@@ -3875,19 +3875,17 @@ default T single(final Predicate<? super T> predicate) {
      */
     default <R> ReactiveSeq<R> retry(final Function<? super T, ? extends R> fn, final int retries, final long delay, final TimeUnit timeUnit) {
```

Comment:
Transformable is anything that has a map (transform) operator, the standard functional term is Functor (but we are reserving that for the purer Functor type class). You can derive retry from map (it's basically retrying the map operation).

We could break it up into an inheritance hierarchy if that makes sense (as long as every method doesn't need to be reimplemented with a type change and super call each time).